### DISCUSSION TITLE
Is it OK message for fwupd? "FuUsbDevice          failed to load BOS descriptor from USB device"

### DISCUSSION THREAD

**Discussion by necros2k7** [fwupd/fwupd#6907] [Q&A]
": USB error on device 1022:15b7 : Operation not supported or unimplemented on this platform [-12]"
1.9.15 win x64

**Comment by hughsie** [comment]
You can ignore this I think. Maybe https://github.com/fwupd/fwupd/pull/6908 will help here.
